mobile/lib/state/audio_handler_cubit.dart

```diff
@@ -1,22 +1,28 @@
 import 'package:audio_service/audio_service.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 
-import '../components/radio_player/audio_player_handler.dart';
+import 'package:radioaktywne/components/ra_player/ra_player_handler.dart';
+import 'package:radioaktywne/components/ra_player/ra_player_recources.dart';
 
-class AudioHandlerCubit extends Cubit<AudioHandler?> {
+class AudioHandlerCubit extends Cubit<RaPlayerHandler> {
   /*
   initializing AudioHandlerCubit with null because proper init is asynchronous,
   but cubit init has to be synchronous, so it needs something to store
    */
-  AudioHandlerCubit() : super(null) {
-    initAudioHandler(); // proper 'init function' call
+  AudioHandlerCubit({required MediaItem initialMedia})
+      : super(
+          RaPlayerHandler(mediaItem: RaPlayerConstants.radioMediaItem),
+        ) {
+    initAudioHandler(initialMedia);
   }
 
-  Future<void> initAudioHandler() async {
+  Future<void> initAudioHandler(MediaItem initialMedia) async {
     final audioService = await AudioService.init(
-      builder: AudioPlayerHandler.new,
+      builder: () => RaPlayerHandler(mediaItem: initialMedia),
       config: const AudioServiceConfig(
         androidNotificationChannelName: 'Live radio stream',
+        // androidNotificationChannelId: 'pl.radioaktywne.channel.audio',
+        // androidNotificationChannelId: 'com.ryanheise.bg_demo.channel.audio',
```

Comment:
what are these about? add comment to clarify if it's gonna be useful in the future